mitogen/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: add EPEL repo on CentOS
+  yum:
+    name: epel-release
+    state: present
+  when: ansible_distribution == 'CentOS' and ansible_distribution_major_version == '7'
+  tags: [ mitogen, epel ]
+
+- name: ensure pip is installed
```

Comment:
Would using the pip ansible-role as a dependency make sense?

Reply:
> Would using the pip ansible-role as a dependency make sense?
Thanks for explaining this...I will make this a dependency.

Reply:
Added pip install via meta